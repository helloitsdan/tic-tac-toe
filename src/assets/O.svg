<svg title="O" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
  <circle class="line" cx="50.81" cy="51.81" r="42" fill-rule="evenodd" clip-rule="evenodd" fill="none" stroke-width="8" stroke-linecap="round" stroke-miterlimit="10"/>
</svg>
