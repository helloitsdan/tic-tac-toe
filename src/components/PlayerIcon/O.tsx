import React, { FunctionComponent } from 'react'
import { ReactComponent as OIcon } from '../../assets/O.svg'

const O: FunctionComponent = () => (
  <OIcon title="O" className="c-player-piece" />
)

export default O
