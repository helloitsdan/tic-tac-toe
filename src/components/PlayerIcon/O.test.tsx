import React from 'react'
import { render } from '@testing-library/react'

import O from './O'

describe('O', () => {
  it('renders', () => {
    const { asFragment } = render(<O />)
    expect(asFragment()).toMatchSnapshot()
  })
})
