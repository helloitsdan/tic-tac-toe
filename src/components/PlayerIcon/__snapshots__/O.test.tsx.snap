// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`O renders 1`] = `
<DocumentFragment>
  <svg
    class="c-player-piece"
    title="O"
  >
    O.svg
  </svg>
</DocumentFragment>
`;
